### ISSUE TITLE
Sensor mount Day #4

### ISSUE THREAD

**Issue by WROMagnet** [WROMagnet/Magnet-Robots#40]
Today, March 27, 2024, we are going to mount the ultrasonic sensors.

Today the goal is to assemble the ultrasonic sensors and place them where they go, but first we have to make sure that the sensors we have work. We have 4 ultrasonic sensors and additionally we have two color sensors. We will be testing the sensors and then we will be mounting them on the robot.

**Comment by WROMagnet** [comment]
We are testing the sensors and one of the sensors was not working but we knew why it was not working and after 2 minutes we found the problem with the sensor. The sensor was not working because one of its capacitors was exploded. Here we have an image of the capacitor.



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/91aaf96e-9b96-471a-81e8-130225694a08)

**Comment by WROMagnet** [comment]
We are looking for a sensor to replace the damaged sensor and we will now be mounting the sensors on the robot.

- We made a plan marking where we want the sensors on the robot.



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/fab16232-a175-409a-bf91-4251f9c2a655)

**Comment by WROMagnet** [comment]
- We already mounted the ultrasonic sensors and additional color sensors but we had some complications when assembling them because we had to use additional feet for the mount like the one from the Pitsco kit and we placed them so that it does not detect any involuntary part of the robot. Now We will be connecting the sensors to the controller (PRIZM) through some cables.

- We have an idea of ​​how to pass the sensor cables to the PRIZM. The idea is to make a hole where we can pass the cables to the second floor where the PRIZM is. Let's plow the hole with a drill and so that it cannot be seen we will drill it under the PRIZM.

- We have already finished mounting the sensors and we have already connected the sensors to the PRIZM and with this we conclude this day.
